amd_openvx/openvx/hipvx/vision_kernels.cpp

```diff
@@ -799,7 +799,7 @@ int HipExec_FastCorners_XY_U8_NoSupression(hipStream_t stream, vx_uint32 capacit
 }
 
 __global__ void __attribute__((visibility("default")))
-Hip_FastCorners_XY_U8_Supression(uint capacityOfDstCorner, char *pDstCorners,
+Hip_FastCorners_XY_U8_Supression(uint capacityOfDstCorner, char *pDstCorners, uint pDstCornersOffset,
```

Comment:
change the pHipDstCornerOffset to cornerBufferOffset here and in all occurrences below